packages/helix-shared-bounce/src/bounce.js

```diff
@@ -53,7 +53,8 @@ function bounce(func, { responder, timeout = 500 }) {
     const signal = timeoutSignal(2 * timeout);
     const actualResponse = (async () => {
       try {
-        return await fetch(request, {
+        return await fetch(request.url, {
+          ...request.init,
```

Comment:
Note that this is a Helix Fetch implementation detail, i.e. non standard and undocumented.

Reply:
Yes, it is the same thing we use in helix-shared-bodydata.